Makefile

```diff
@@ -537,8 +533,8 @@ test-shlib-exports: header-exports.txt shlib-exports-$(SHLIB_FLAVOUR).txt
 	( echo "Error: Found unexported symbols (listed above)" ; false )
 
 # Extract symbols that should be exported from public headers using ctags
-# Filter out macros in htslib/hts_defs.h, and knet_win32_ functions that
-# aren't needed on non-Windows platforms.
+# Filter out macros in htslib/hts_defs.h that aren't needed on non-Windows
+# platforms.
```

Comment:
Suspect “that aren't needed on non-Windows platforms” pertained only to `knet_win32_` and can disappear too.

Reply:
I was unsure of that and ummed and ahhed over it, but happy to remove.  I did start to edit the associated perl script, but figured it's harmless for it to check for functions that don't exist and I'm more likely to break it in doing so.  It's perhaps still something that should be changed though.